src/lib/utils.ts

```diff
@@ -166,6 +166,28 @@ function removeRole(interaction: CommandInteraction,
 	return false;
 }
 
+export function msToHMS(ms: number): string {
```

Comment:
Sorry for just bringing this up now, but doesn't moment have a utility for this right now? Or pretty-ms? (I think [exactly this](https://www.npmjs.com/package/pretty-ms#usage)